src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -486,6 +486,12 @@ void RegisterAppInterfaceRequest::Run() {
         }
         case mobile_apis::AppHMIType::WEB_VIEW: {
           application->set_webengine_projection_enabled(true);
+          const std::string default_webview_layout =
+              application_manager_.ScreenPredefinedLayoutToString(
```

Comment:
@AByzhynar 
Maybe auto ?

Reply:
@SNiukalov Done in c5170197a2d095b02c96492eb6a782f406bb9c36

Reply:
@SNiukalov Done in c5170197a2d095b02c96492eb6a782f406bb9c36